packages/local-cli/runAndroid/runOnAllDevices.js

```diff
@@ -0,0 +1,117 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @flow
+ */
+
+/* eslint-disable consistent-return */
+
+const chalk = require('chalk');
+const { spawnSync, execFileSync } = require('child_process');
+
+const adb = require('./adb');
+const tryRunAdbReverse = require('./tryRunAdbReverse');
+const tryLaunchAppOnDevice = require('./tryLaunchAppOnDevice');
+
+function runOnAllDevices(
+  args,
+  cmd,
+  packageNameWithSuffix,
+  packageName,
+  adbPath
+) {
+  try {
+    const gradleArgs = [];
+    if (args.variant) {
+      gradleArgs.push(
+        `${
+          args.appFolder
+        }:install${args.variant[0].toUpperCase()}${args.variant.slice(1)}`
```

Comment:
This change has a minor breaking change that we should be aware of. I am not sure which approach is the desired one - we should probably check it and decide what is the way we want to keep it.

`args.appFolder` is an optional CLI flag that defaults to "app". Prior to this PR, we were not using this option at all, making it impossible to change the app to install on the device, when multiple are defined. 

The command executed by Gradle was always e.g: "installDebug", where in reality, it should've been "app:installDebug", since the "appFolder" defaults to "app". 

That means, after this PR is merged, all the default executions will contain "app:". This might be breaking for people that were depending on this undocumented bug.

I wonder whether we should stick to it, or remove the default "app" value of "appFolder" and make it either "appFolder:installDebug" or "installDebug", depending on whether it is passed to the CLI.

Reply:
Yea I think it would be better to default to keep running `installDebug` if `appFolder` is not specified.

Reply:
Seems reasonable, I will make that change.